packages/flutter/lib/src/material/text_theme.dart

```diff
@@ -194,20 +343,31 @@ class TextTheme with Diagnosticable {
     TextStyle? button,
     TextStyle? overline,
   }) {
+    assert(
+    (displayLarge == null && displayMedium == null && displaySmall == null && headlineMedium == null &&
```

Comment:
I realize that there's no nice and compact way to format this, but: two-space indent

Reply:
Fixed